### DISCUSSION TITLE
For PHP?

### DISCUSSION THREAD

**Discussion by kargnas** [BoundaryML/baml#693] [Ideas]
Do you have any plan for extending this template language into PHP industry?

**Comment by hellovai** [comment]
We'd love to consider a language. To help us prioritize between multiple languages to support, if you can get 10 more people to upvote PHP, we'd be happy to bump it to the top of the queue.

We are currently planning on Java, Rust, and Go support prior!

**Comment by hellovai** [comment]
@kargnas this is now solved! check out https://docs.boundaryml.com/docs/get-started/quickstart/openapi If you have any questions please llet me know!
